src/main/java/io/github/ust/mico/kafkafaasconnector/configuration/KafkaConfig.java

```diff
@@ -45,4 +64,8 @@
      */
     @NotBlank
     private String deadLetterTopic;
+
+
+    @NotBlank
+    private String filteredTestMessagesTopic;
```

Comment:
I don't think this will improve the name. ´filteredTestMessagesTopic´ implies that only filtered test messages are put into this topic. Your name could be interpreted that all test messages are redirected to it.

Reply:
It would avoid ambiguity with the message filter pattern. As test messages are in fact never filtered but only rerouted/collected. And all test messages should eventually end up in this topic. So all test messages will be redirected to this topic at some point. This eventually is the only thing my suggestion can not capture.